index.js

```diff
@@ -127,25 +145,23 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
+const inventoryCommand = (message) => {
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    const userIdentification = message.author.id;
+    const userInventory = userData[userIdentification].inventory;
+    // let cardIDs = Object.keys(userInventory);
+    let cardData = Object.values(userInventory);
+    let inventoryList = "";
 
+    cardData.forEach(function (item) {
+      inventoryList += "•  " + item.name + " \n";
+    });
```

Comment:
you can use ES6 arrow notation to shorten a little:
```js
    cardData.forEach((item) => {
      inventoryList += "•  " + item.name + " \n";
    });
```

We could actually go even further if we only have 1 line in the function:
```js
cardData.forEach(item => inventoryList +=  "•  " + item.name + " \n");
```

This is similar to how braces are optional for 1 line if statements, e.g.

```js
if (a === true)
   console.log("hi");
```

is the same as 

```js
if (a=== true) {
  console.log("hi");
}
```